zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/RestoreFromBackupTool.java

```diff
@@ -618,4 +658,26 @@ private void copyProcessedRestoredFilesToDestination(FileTxnSnapLog restoreTempS
       Files.copy(processedFile.toPath(), new File(finalDestinationBase, fileName).toPath());
     }
   }
+
+  /**
+   * If the CLI command has specified a znode path to perform spot restoration,
+   * run the spot restoration tool on that path
+   * @throws IOException
+   */
+  @VisibleForTesting
+  protected void performSpotRestorationIfConfigured() throws IOException {
+    if (znodePathToRestore != null) {
```

Comment:
What about the `else` condition?

Reply:
`else` means this command is not meant for a spot restoration, only for an offline restoration, so nothing will be done here.

Reply:
In that case, should we log/print it?

Reply:
Added logging.